packages/api/src/elastic/pages.ts

```diff
@@ -28,14 +28,21 @@ import {
 } from './types'
 
 const appendQuery = (builder: ESBuilder, query: string): ESBuilder => {
-  const fields = ['title', 'content', 'author', 'description', 'siteName']
+  const fields = [
+    { field: 'title', boost: 3 },
+    { field: 'content', boost: 1 },
+    { field: 'author', boost: 1 },
+    { field: 'description', boost: 1 },
+    { field: 'siteName', boost: 1 },
```

Comment:
We can experiment, but I suspect we want to boost title even more and maybe siteName a little

Reply:
I have tuned a bit the boost score and the title boost score seems good enough.

I now boost the siteName score by 2 and it looks good.
https://github.com/omnivore-app/omnivore/pull/2457#issuecomment-1617799805